src/core/Toast/Toaster.tsx

```diff
@@ -18,6 +40,22 @@ export type ToastOptions = Omit<
 export default class Toaster {
   private toasts: ToastProps[] = [];
   private lastId = 0;
+  private settings: ToasterSettings = {
+    order: 'descending',
+    placement: 'top',
+  };
+
+  /**
+   * Set global Toaster settings for toasts order and placement.
+   * Settings will be applied to all existing and new toasts on the page.
+   */
+  public setSettings(settings: ToasterSettings) {
+    settings.placement ??= this.settings.placement;
+    settings.order ??= settings.placement?.startsWith('bottom')
+      ? 'ascending'
+      : 'descending';
+    this.settings = settings;
+  }
 
   public positive(content: React.ReactNode, settings?: ToastOptions): void {
```

Comment:
Maybe rename `settings` to `options` here, to prevent confusion.

Not sure we're gonna avoid confusion anyhow, but still.

Reply:
Renamed ToastOptions var to options